src/training_hub/algorithms/lora_grpo.py

```
@@ -736,18 +737,22 @@ def _subprocess_entry(algorithm_params, results_path, error_path):
             )
         except SystemExit:
             pass  # os._exit from shutdown — results saved in _run_training
-        except Exception as e:
+        except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)